### DISCUSSION TITLE
When user reset password it should reset password of username from all realm

### DISCUSSION THREAD

**Discussion by Bhavik237** [keycloak/keycloak#21673] [Q&A] [answered]
Hello everyone,

I'm looking for ways to reset a user's password across all realms when the username is the same. Specifically, I need to initiate the password reset process when a user ask for reset password .

Your input and suggestions would be greatly appreciated. Thank you!

**Comment by christian-2** [comment, marked as answer]
I don't think this is possible in Keycloak: each realm manages its own users and is not supposed to interfere with its siblings. 

Presumably you trigger the password reset by requesting some URL from JavaScript, or similar: if so, you could install a handler at the URL that walked all realms and did the desired change there via the REST API. Perhaps not the most elegant or simplest way, but the only one I am aware of.

**Reply by Bhavik237** [reply]
okay , I will try it.
